### ISSUE TITLE
Why freeze batch norm parameters when training?

### ISSUE THREAD

**Issue by leao1995** [kuangliu/pytorch-retinanet#18]

**Comment by xichangzun** [comment]
it's a common practice.First, Because the pretrain network's bn layers have been trained. Second,Object Detection 's batchsize is small, hard to make bn parameter stable.

**Comment by prakashjayy** [comment]
use Group norm instead of batch norm . it is more stable.

**Comment by lxtGH** [comment]
Use synchronized batch normalization

**Comment by PhilipMay** [comment]
> Use synchronized batch normalization

Using sync batch norm does not help with single GPU training and low batch sizes though.
